lib/mysql_rewinder/cleaner.rb

```diff
@@ -21,7 +21,7 @@ def clean(tables:)
       target_tables = (tables - @except_tables) & all_tables
 
       if target_tables.empty?
-        @logger&.debug { "[MysqlRewinder][#{@db_config[:database]}] Skip DELETE query because target_table is empty." }
+        @logger&.debug { "[MysqlRewinder][#{@db_config[:database]}] Skip DELETE query because target_table is empty. target tables: #{tables}, except tables: #{@except_tables}, all tables: #{all_tables}." }
```

Comment:
We must log the exact variable names

```suggestion
        @logger&.debug { "[MysqlRewinder][#{@db_config[:database]}] Skip DELETE query because target_table is empty. tables: #{tables}, @except_tables: #{@except_tables}, all_tables: #{all_tables}." }
```